.devcontainer/devcontainer.json

```diff
@@ -21,8 +21,7 @@
 			]
 		}
 	},
-	"containerEnv": {
-		"PALM_APIKEY": "<YOUR_PALM_API_KEY>"
+  "containerEnv": {
```

Comment:
Did you intend to swap out the palm for local model or actually give user an option to use local model to generate the embeddings?